jekyll/_cci2/testing-ios.md

```diff
@@ -88,6 +90,38 @@ If upgrading from Cocoapods 1.7 or older, additionally ensure the following line
 source 'https://github.com/CocoaPods/Specs.git'
 ```
 
+#### Homebrew
+
+Homebrew, by default, will check for updates at the start of any operation. As Homebrew has a fairly frequent release cycle, this means that the step execution can take some extra time to complete.
+
+If build speed, or bugs introduced by new Homebrew updates, are of a concern, this can be disabled. On average, this can save 2-5 minutes per job.
```

Comment:
```suggestion
If build speed or bugs introduced by new Homebrew updates are a concern, this update-check feature can be disabled. On average, this can save 2-5 minutes per job.
```